src/dcommands/names.js

```diff
@@ -8,18 +8,72 @@ class NamesCommand extends BaseCommand {
             category: bu.CommandType.GENERAL,
             usage: 'names [user] [flags]',
             info: 'Returns the names that I\'ve seen the specified user have in the past 30 days.',
-            flags: [{ flag: 'a', word: 'all', desc: 'Gets all the names.' },
-            {
-                flag: 'v',
-                word: 'verbose',
-                desc: 'Gets more information about the retrieved names.'
-            }]
+            flags: [
+                {
+                    flag: 'a',
+                    word: 'all',
+                    desc: 'Gets all the names.'
+                },
+                {
+                    flag: 'v',
+                    word: 'verbose',
+                    desc: 'Gets more information about the retrieved names.'
+                },
+                {
+                    flag: 'r',
+                    word: 'remove',
+                    desc: 'Removes all your usernames from the database.'
```

Comment:
Misleading description
```suggestion
                    desc: 'Removes the specified username from the database, or all usernames if used with the `all` flag.'
```